dotenv-sample.env

```diff
@@ -0,0 +1,7 @@
+BACKEND_TOKEN="really really really really really really really long secret"
+# local dev server
+SERVER_HOST=http://127.0.0.1:8001/
+# local job-server
+API_SERVER=http://127.0.0.1:8000/
+# workspaces dir
+WORKSPACES=./workspaces
```

Comment:
Could you pop a line before each comment? Helps to break up the text for anyone who struggles with lines merging 🙂